extensions/data-plane-selector/selector-spi/src/main/java/org/eclipse/dataspaceconnector/dataplane/selector/store/DataPlaneInstanceStore.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright (c) 2020 - 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.dataplane.selector.store;
+
+import org.eclipse.dataspaceconnector.dataplane.selector.instance.DataPlaneInstance;
+
+import java.util.Collection;
+import java.util.stream.Stream;
+
+/**
+ * Holds all {@link DataPlaneInstance} objects that are known to the DPF selector.
+ * The collection of those objects is not regarded immutable, so adding at runtime is possible, implementors need to be
```

Comment:
Super nit, I think something like the following would be clearer:
The collection of {@link DataPlaneInstance} objects is mutable at runtime, so implementations must take that into account.